php/class-post-type.php

```diff
@@ -154,7 +154,6 @@ public function setup_metaboxes() {
 	 */
 	public function remove_publish_metabox() {
 		remove_meta_box( 'slugdiv', static::SLUG, 'normal' );
-		remove_meta_box( 'submitdiv', static::SLUG, 'side' );
```

Comment:
Keep removing if the snapshot's associated theme is not equal to the current theme.